text/0000-rfc-process.md

```diff
@@ -0,0 +1,238 @@
+- Copy `0000-template.md` to `text/0000-my-feature.md` (where 'my-feature' is descriptive. don't assign an RFC number yet).
+- Fill in RFC. Any section can be marked as "N/A" if not applicable.
+- Submit a pull request.
+
+#### Finding a Steward
+All RFCs, even project RFCs "belong" to a team. For project RFCs, The team lead of the responsible team is the steward of the RFC. For team RFCs any maintainer may be the steward. The author and the steward of an RFC may, at times, be the same person.
+
+In many but not all cases the appropriate team to own an RFC will be obvious. When the appropriate team is not obvious, the author should work with the community to find a home within one of the teams. Factors to consider when finding home include:
+* Which team has the most relevant technical context?
+* Which team has the deepest understanding of the use-case and the needs of impacted personas?
+* Which team is responsible for the components most impacted by the proposed change?
+* Which team is enthusiastic about supporting the change?
+
+If a home cannot be found for a draft RFC it remains in draft until one can be found. This does not necessary mean that the RFC is not a good idea or isn't something the project will take up eventually. It may simply be that the project does not have the bandwidth to prioritize this particular change at this particular time.
+
+#### Stewardship
+
+The steward and their team should:
+ * work with the author of the RFC to ensure that the RFC is complete and implementable contingent upon approval. This can happen synchronously at team working groups or asynchronously via github and slack.
+ * raise visibility to and solicit feedback from other stakeholders including the TOC, other teams, and the community at large. This can happen synchronously at the project working group and asynchronously via github and slack.
+ * drive consensus for the RFC by incorporating feedback from stakeholders so that the RFC has the best possible chance of approval during the voting process.
+ * ensure there is a plan in place to implement the RFC in a reasonable time frame, contingent upon approval. The team itself need not implement the RFC but we should not approve RFCs for which we have no concrete plan to implement.
+ * in the case of complicated or risky RFCs, a POC should be developed at this stage to validate and de-risk the proposed design.
+
+#### Voting
+
+Ideally the voting should be a formality and not a moment to discover new disagreement, consensus already haven been driven by the steward.
+
+When the steward deems the RFC ready and likely to be accepted they should formally call for votes and set an end date for voting. This process does not prescribe a length for the voting window, but stewards should make a good faith effort to ensure that all interested parties, and in particular those with binding votes, have adequate opportunity to review the finalized RFC and cast votes.
```

Comment:
Can we set a minimum time period of a week for RFCs unless all of the binding votes are cast? We should also get rid of FCP and replace it with this voting period.